src/lang/en.json

```diff
@@ -132,10 +133,7 @@
         "tableTooSmall": "Minimum size of the table is 1x1",
         "emptyModel": "Model table is required",
         "emptyModelInput": "Make sure you have filled all table fields",
-        "titleTooLong": "Maximum 64 characters",
-        "invalidTag": "Invalid tag: only letters, numbers and \"-\" symbols are allowed",
```

Comment:
"invalidTag": "Invalid tag: only letters, numbers and \"-\" symbols are allowed",	
"invalidTagLength": "Invalid tag: tag is too short",	
"invalidTagCount ": "Maximum tag limit is 5" deleted. Why?